webhook/certificates/certificates_test.go

```diff
@@ -176,9 +167,20 @@ func TestReconcileMakeSecretFailure(t *testing.T) {
 		Key:     key,
 		Objects: []runtime.Object{secret},
 	}, {
-		Name:    "secret does not exist",
+		Name:    "malformed secret",
 		Key:     key,
 		WantErr: true,
+		Objects: []runtime.Object{&corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      secretName,
+				Namespace: system.Namespace(),
+			},
+			Data: map[string][]byte{
+				// certresources.ServerKey:  []byte("missing"),
```

Comment:
rm ?

Reply:
When I initially wrote these I had intentionally left them in to make clear which keys were excluded where.  See also the content `"present"` vs. `"missing"`.

Reply:
We can convey this other ways, but we should fix the whole file. 😇